app/controllers/api/databases.php

```diff
@@ -381,6 +383,18 @@ function updateAttribute(
     return $attribute;
 }
 
+App::init()
+    ->groups(['api', 'database'])
```

Comment:
why's this applied specifically to the `database` group ?  shouldn't `api` already cover `databases` as well?

Reply:
Also we have other `get` and `list` endpoints in other services which should also have the timeouts